ocamltest/run_stubs.c

```diff
@@ -80,5 +82,7 @@ CAMLprim value caml_run_command(value caml_settings)
   settings.logger = logToChannel;
   settings.loggerData = Channel(Field(caml_settings, 7));
   res = run_command(&settings);
+  for (p = *(settings.argv); p!=NULL; p++)
+    caml_stat_free(p);
```

Comment:
You should also free `settings.argv` itself, right ? (compare with https://github.com/ocaml/ocaml/pull/1357/files#diff-9861eb39c9e039fa7cb0230fc8e2a6ca).